pkg/scalers/kafka_scaler.go

```diff
@@ -108,8 +110,10 @@ func parseKafkaMetadata(resolvedEnv, metadata, authParams map[string]string) (ka
 
 	meta.authMode = kafkaAuthModeForNone
 	if val, ok := authParams["authMode"]; ok {
+		val = strings.TrimSpace(val)
```

Comment:
were you doing `echo 'value' | base64` by any chance?

Reply:
Yes I am.. Is there another recommended way to do base64 encoding?

Reply:
`echo` just appends a new line character by default. Try `echo -n 'value' | base64` instead to tell it not to do that.

Reply:
I will make sure to mention this in the documentation.